### PR TITLE
Fixed the cross origin problem

### PR DESCRIPTION
**Pull request by antrromet**
Hey @smadha 
So I looked online and its a common problem. I used jquery ajax call now instead of XML request. Its working fine now.

### CODE REVIEW

**Comment by antrromet** [comment]
@smadha In this though, I've added a failsafe. So if theres an error by any chance, it will show the default solr link.

File: geoparser_app/static/js/app.js [hunk 1]
```diff
@@ -600,6 +614,54 @@ $(function() {
 			})
 
 })
+
+$(function() {
+    $("#viewIndexResults").keypress(function(e) {
+        if(e.which == 13) {
+            $("#searchResults").empty();
+            var keyword = $(this).val();
+            var searchLink = "http://localhost:8984/solr/scientific_content_enrichment/select?indent=on&q="+keyword+"&wt=json";
```
Comment (smadha) [inline geoparser_app/static/js/app.js]: Looks like your local change. May be you can do it in a generic way by showing dropdowns of available indexes as we do currently with view index button.

File: geoparser_app/static/js/app.js [hunk 2]
```diff
@@ -600,6 +614,54 @@ $(function() {
...
+            
+            var xmlhttp = new XMLHttpRequest();
```
Comment (smadha) [inline geoparser_app/static/js/app.js]: jquery ajax
jsonp

**Comment by antrromet** [comment]
@smadha Ahh my bad! I didn't mean to push that in the PR. That was just for the demo for the assignment that I was working on!

**Comment by smadha** [comment]
Hey @antrromet as you see I already did the failsafe part so we don't need that. But I see you have also handled cross domain error by using jsonp. Can you please ensure everything works fine after I merge this PR to master. I see you merged latest changes in your PR but still want to double check.

**Comment by antrromet** [comment]
Hey @smadha, I just checked and I think its fine now. I also removed the search code for now. I will add a list of dropdown and then submit a PR for that later on.

**Comment by chrismattmann** [comment]
Guys I want @antrromet search code to be added have seen it and it's great cam we integrate it

**Pull request merged**